src/views/CardStack.js

```diff
@@ -65,7 +69,7 @@ type Props = {
 /**
  * The duration of the card animation in milliseconds.
  */
-const ANIMATION_DURATION = 200;
+const ANIMATION_DURATION = 250;
```

Comment:
can you describe how you came to this number? Ideally it would be via measurement, but you can also say 'approximation'